index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Trackomatic Test</title>
+  </head>
+  <body>
+    <!-- Google Analytics -->
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', { trackingId: 'UA-1980766-29', cookieDomain: 'auto', name: 't1' });
+      ga('t1.require', 'trackomatic', { debug: 1 });
+      ga('t1.send', 'pageview');
+    </script>
+
+
+    <a href="#foo">Should not fire</a>
+    <a href="http://viget.com#bar">Should fire</a>
+    <a href="http://pinterest.com">Should fire</a>
+    <a href="http://viget.com/sample.pdf">Should fire</a>
+    <a href="mailto:albert.wavering@viget.com">Should fire</a>
+    <a href="tel:+7038910670">Should fire</a>
```

Comment:
I had a doc just like this at one point, but it also included some dynamic links so I could test the delegation. Maybe worth adding?

https://gist.github.com/averyvery/8bc832d89e2043ac73a2#file-delegated-click-tracking-html-L15